src/systemtest/resources/features/ArtifactFlow.feature

```diff
@@ -11,36 +11,35 @@ Feature: Artifact system test
     And a jenkins job "TCSTCFJob" from "src/systemtest/resources/JenkinsShellScripts/TCSTCFScript.txt" is created with parameters: EVENT_ID,
     And a jenkins job "CLMJob" from "src/systemtest/resources/JenkinsShellScripts/CLMScript.txt" is created with parameters: EVENT_ID,
     And a jenkins job "ArtPJob" from "src/systemtest/resources/JenkinsShellScripts/ArtPScript.txt" is created with parameters: EVENT_ID,
-    And a jenkins job "FlowCompleteJob" from "src/systemtest/resources/JenkinsShellScripts/FlowCompleteScript.txt" is created with parameters: EVENT_ID,
+    And a jenkins job "FlowCompleteJob" from "src/systemtest/resources/JenkinsShellScripts/FlowCompleteScript.txt" is created with parameters: ,
```

Comment:
Gurka statements should not end with , or .?

Reply:
Actually this has to. It's a workaround to use lists on one line. The empty lines need some text after ":" and "," is the seperator which means it won't be interpreted as a parameter. I can remove them from the lines with parameters but i'd rather keep them on all lines in that case.

Reply:
I would prefer that parameters are added in it´s own line, but since this is already merged I guess it would require too much changes to implement and this may be ok.

Reply:
I think that was to cluttered and it did not work to send 0 parameters in that case.